src/main/java/fact/TriggerEmulation/Discriminator.java

```diff
@@ -0,0 +1,120 @@
+package fact.TriggerEmulation;
+
+import fact.Constants;
+
+/**
+ *Class with functions to discriminate the signals of a given patch (or a bunch of them).
+ * Furhtermore it contains helper functions for the discrimination e.g. conversion between DAC and mV
+ */
+public class Discriminator {
+
+    public static int default_slice = 0;
+
+    /**
+     *Discriminate the signal of a given patch
```

Comment:
maybe explain better what the return value of this function is ... maybe like:
"""
Return the slice (index of data) of the first occurence of:
  * signal is over threshold *and*
  * signal is over threshold for a long enough time.
"""
or so.